_posts/2017-01-02-metting-pre-first-release.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Meeting pre first release"
+date: 2017-01-02 08:02:58
+categories: code
+---
+
+David Durieux, Frederic Mohier and Sebastien Coavoux met the 29 December 2016 to discuss about the first release for Alignak, the version 1.0.0 !
+
+We have been received at the DCS company office at Lyon in France (DCS is our mainly partner, see [friends page](/ourfriends)).
+
+
+The meeting was about the first release of Alignak and the review is:
+
+* fix packaging problems for Alignak and the check packs, modules
+* take decision about some issues and simplification of code
+* decided for the future (version 2.0 or after) to use backend for all configuration and drop the cfg files support
```

Comment:
*decide* instead of *decided*
*to use the backend*
*drop the configuration based on flat files support*